### PR TITLE
975 change font size and color

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: static/main.css [hunk 1]
```diff
@@ -25,6 +25,61 @@ h1, h2, h3, h4, h5, h6 {
   color: #444444;
 }
```
Comment (minump) [inline static/main.css:27]: Maybe delete these lines originally present ?
h1, h2, h3, h4, h5, h6 {
  color: #444444;
}
h2 {
  text-align: center;
}

File: static/main.css [hunk 2]
```diff
@@ -25,6 +25,61 @@ h1, h2, h3, h4, h5, h6 {
...
+  font-family: proxima-nova,sans-serif;
+  font-size: 16px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.25;
+  letter-spacing: normal;
+  color: #404040;
+}
+
```
Comment (minump) [inline static/main.css:82]: Should the styling be removed from the respective tags in the html file? For example : 
Remove ```<h5 style="float:left;margin-top: 13px;margin-left: 5px;padding-right: 18px">Group</h5>``` styling from user-events.html?

**Review by minump — COMMENTED** [review]
reviewed

File: static/main.css (hunk 1 above)
Comment (bingzhang) [reply static/main.css]: removed

File: static/main.css (hunk 2 above)
Comment (bingzhang) [reply static/main.css:78]: move in css.

**Comment by minump** [comment]
I think the title "User Events" needs to go up a little. There seems to be a lot of space between the header and the title. I think the title now is inline with "Group" drop-down menu. Also the "Events Per page" drop down has a different sized box. 
Just throwing these in as comments as it is not related to this PR. And redesigning an already existing page and be messy and might get nice-looking only with multiple iterations.

Screen Shot 2022-11-15 at 4 27 15 PM

**Comment by minump** [comment]
Tested and approved.

**Comment by bingzhang** [comment]
> I think the title "User Events" needs to go up a little. There seems to be a lot of space between the header and the title. I think the title now is inline with "Group" drop-down menu. Also the "Events Per page" drop down has a different sized box. Just throwing these in as comments as it is not related to this PR. And redesigning an already existing page and be messy and might get nice-looking only with multiple iterations.
> 
> Screen Shot 2022-11-15 at 4 27 15 PM

Thanks, @minump. I think we can tweak every page later after we merge all the pull requests. Since it will change after we merge prs.

**Comment by minump** [comment]
> > I think the title "User Events" needs to go up a little. There seems to be a lot of space between the header and the title. I think the title now is inline with "Group" drop-down menu. Also the "Events Per page" drop down has a different sized box. Just throwing these in as comments as it is not related to this PR. And redesigning an already existing page and be messy and might get nice-looking only with multiple iterations.
> > Screen Shot 2022-11-15 at 4 27 15 PM
> 
> Thanks, @minump. I think we can tweak every page later after we merge all the pull requests. Since it will change after we merge prs.

yup. sounds like a plan Bing.
Approving the PR.

**Pull request merged**